pytorch_blade/pytorch_blade/compiler/jit/torch/shape_analysis.cpp

```diff
@@ -104,6 +104,27 @@ c10::Device getDeviceFromValue(Value* value) {
   return device;
 }
 
+c10::optional<at::ScalarType> getScalarTypeFromValue(Value* value) {
+  c10::optional<at::ScalarType> scalarType;
+  if (value == nullptr)
+    return scalarType;
+  if (value->node()->kind() == prim::dtype) {
```

Comment:
Nit: add a shape inference test for `prim::dtype`

Reply:
OK，both torchscript and e2e tested